### PR TITLE
New mapping + controller detection

### PR DESCRIPTION
**Pull request by lee17017**
Controll changes:
Right trigger: attack
Left trigger: throw
Right + Left Bumper: rotate 
X: Dig

The InputHandler detects if a xbox controller is plugged in or not at startup time. And also if the controller is the speedlink controller of Konstantin.

### CODE REVIEW

File: Assets/Scripts/PlayerInput/InputDictionary.cs [hunk 1]
```diff
@@ -19,38 +19,39 @@ public static class InputDictionary
         #region Xbox
         public const string XboxLeftJoystickHorizontal = "XboxLeftJoystickHorizontal";
         public const string XboxLeftJoystickVertical = "XboxLeftJoystickVertical";
-
+        public const KeyCode XboxSpeedLinkLeftTrigger = KeyCode.Joystick1Button7;
```
Comment (EnigmaticDragon) [inline Assets/Scripts/PlayerInput/InputDictionary.cs]:
```suggestion
        public const KeyCode XboxSpeedLinkLeftTrigger = KeyCode.Joystick1Button6;
```

File: Assets/Scripts/PlayerInput/InputDictionary.cs [hunk 2]
```diff
@@ -19,38 +19,39 @@ public static class InputDictionary
...
+        public const KeyCode XboxSpeedLinkRightTrigger = KeyCode.Joystick1Button8;
```
Comment (EnigmaticDragon) [inline Assets/Scripts/PlayerInput/InputDictionary.cs]:
```suggestion
        public const KeyCode XboxSpeedLinkRightTrigger = KeyCode.Joystick1Button7;
```

File: Assets/Scripts/PlayerInput/XboxInput.cs [hunk 3]
```diff
@@ -26,9 +27,32 @@ public class XboxInput : InputMethod
 
         private const float maxAxisValue = 0.07537688f;
 
+        private bool speedLinkController = false;
+        private void Start()
+        {
+            if (Input.GetJoystickNames().Count() > 0 && Input.GetJoystickNames()[0] == "SPEEDLINK TORID")
+            {
+                speedLinkController = true;
+
```
Comment (EnigmaticDragon) [inline Assets/Scripts/PlayerInput/XboxInput.cs]:
```suggestion
SetSpeedLinkMapping();
```

**Comment by lee17017** [comment]
LeftTrigger is now 6 and right is 7

**Pull request merged**